resources/fetchers/elb_fetcher.go

```diff
@@ -51,7 +50,7 @@ type ELBFetcherConfig struct {
 	KubeConfig                    string `config:"Kubeconfig"`
 }
 
-type LoadBalancersDescription elasticloadbalancing.LoadBalancerDescription
+type LoadBalancersDescription types.LoadBalancerDescription
```

Comment:
Can we replace `types.LoadBalancerDescription` with `awslib.***`? 

Same way as you did for EKS fetcher and the IAM fetcher?